mpi-proxy-split/p2p_drain_send_recv.cpp

```diff
@@ -115,48 +116,46 @@ recvMsgIntoInternalBuffer(MPI_Status status, MPI_Comm comm)
 }
 
 // Go through each MPI_Irecv in the g_async_calls map and try to complete
-// the MPI_Irecv before checkpointing.
+// the MPI_Irecv and MPI_Isend before checkpointing.
 int
-completePendingIrecvs()
+completePendingP2pRequests()
 {
   int bytesReceived = 0;
   dmtcp::map<MPI_Request, mpi_async_call_t*>::iterator it;
   for (it = g_async_calls.begin(); it != g_async_calls.end();) {
     MPI_Request request = it->first;
     mpi_async_call_t *call = it->second;
-    if (call->type == IRECV_REQUEST) {
-      int flag = 0;
-      MPI_Status status;
-      MPI_Test_internal(&request, &flag, &status, false);
-      if (flag) {
-        UPDATE_REQUEST_MAP(request, MPI_REQUEST_NULL);
+    int flag = 0;
+    MPI_Status status;
+    MPI_Test_internal(&request, &flag, &status, false);
+    if (flag) {
+      if (call->type == IRECV_REQUEST) {
         int size = 0;
         MPI_Type_size(call->datatype, &size);
         int worldRank = localRankToGlobalRank(status.MPI_SOURCE,
-                                              call->comm);
+            call->comm);
```

Comment:
This is minor, but I find that the code is more readable when this argument `call->comm` is indented under the first argument (under `status`), as it was before.